scripts/validation/validate.js

```
@@ -132,6 +132,7 @@ let servicesToValidate = args.filter(arg => !arg.startsWith('--'));
     run();
   } catch (error) {
     console.error(error);
+    process.exit(1);
```

Why not just let the error be thrown, then? 🤔